economy/rps.go.tmpl

```diff
@@ -0,0 +1,36 @@
+{{/*
+	Rock-Paper-Scissor game
+	Usage: "-rps"
+ 
+	Recommended trigger: rps
+	Trigger type: Command
+ 
+	Credits:
+	Made by Jordi
+*/}}
+{{/* Configuration variables start */}}
+{{ $name := "coins" }} {{/* the name of the currency */}}
+{{ $bet := 75 }} {{/* amount to bet */}}
+{{ $cooldown := 120 }} {{/* cooldown in seconds */}}
+{{/* Configuration variables end */}}
+
+{{ $var := randInt 3 }}
+{{ $newcoins := sub ((dbGet .User.ID "coins").Value) $bet }}
```

Comment:
{{ $newcoins := sub ((dbGet .User.ID $name).Value) $bet }}

Reply:
no no no, the complete system is based on db entry "coins" for now, ill keep it like that, just to prevent issues